packages/gstreamer.rb

```diff
@@ -3,31 +3,32 @@
 class Gstreamer < Package
   description 'GStreamer is a library for constructing graphs of media-handling components.'
   homepage 'https://gstreamer.freedesktop.org/'
-  @_ver = '1.20.0'
-  version @_ver
+  @_ver = '1.20.3'
+  version "#{@_ver}-1"
```

Comment:
Why do we need the `-1` with a new version?

Reply:
So it doesn't conflict with the version with the other libjpeg I built on my test system...